src/forecast&highlights/WindDirection.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+export function WindDirection({ currentWeather }) {
+  return (
+    <div className="highlight-additional">
+      <span
+        className="material-symbols-outlined wind-direction-icon"
+        style={{
+          transform: `rotate(${currentWeather.wind.deg}deg)`,
+        }}
+      >
+        navigation
+      </span>
+      {"  "}
+      {degreesToText(currentWeather.wind.deg)}
+    </div>
+  );
+}
+
+function degreesToText(deg) {
```

Comment:
Refactor it, consider sending it to ChatGPT

Prompt: "refactor it for better readability"